packages/python-sqlparse/python-sqlparse.spec

```diff
@@ -70,6 +67,9 @@ set -ex
 * Tue Nov 09 2021 Odilon Sousa <osousa@redhat.com> - 0.4.2-1
```

Comment:
I think *this* one needs to be bumped to 0.4.2-2, as it's actually been updated in 3.16, the others need just be tagged